### PR TITLE
Tell AutoDoc to process the bibliography.

### PR DESCRIPTION
**Pull request by jamesjer**
I tried to build version 0.6.12 for Fedora Rawhide, with gap 4.10.2 and AutoDoc 2019.07.24, and saw this in the build logs:

```
#I Producing the index . . .
#I Writing bibliography . . .
Error, Record Element: '<rec>.bibentries' must have an assigned value in
  need := List( r.bibentries, function ( a )
        return RecBibXMLEntry( a, "Text", r.bibstrings );
    end ); at /usr/lib/gap/pkg/GAPDoc/lib/GAPDoc2Text.gi:597 called from 
GAPDoc2TextProcs.(r.name)( r, str 
 ); at /usr/lib/gap/pkg/GAPDoc/lib/GAPDoc2Text.gi:380 called from
GAPDoc2Text( r, path ) at /usr/lib/gap/pkg/GAPDoc/lib/Make.g:48 called from
CallFuncList( makeDocFun, args ); at /usr/lib/gap/pkg/AutoDoc/gap/Magic.gi:575 called from
<function "AutoDoc">( <arguments> )
 called from read-eval loop at makedoc.g:21
you can 'return;' after assigning a value
gap> brk> Reading file "makedoc.g" has been aborted.
```

This commit tells AutoDoc to process the bibliography, thereby fixing the problem.  Fix tested in Fedora Rawhide.

### CODE REVIEW

**Review by fingolfin — APPROVED** [review]
Looks good to me, thanks. Though @james-d-mitchell is the package maintainer, who should approve and merge this :-)

**Comment by fingolfin** [comment]
I was also confused how this could have slipped through, andI think the problem is twofold: one, locally the `.bib.xml` file probably already existed from a previous successful `makedoc.g` run, masking the issue; and two, the CI tests do not test `makedoc.g`. I already noticed this as a problem for `recog`, and added a suitable test for that there. I guess a similar thing should be done in https://github.com/gap-system/pkg-ci-scripts so that all GAP packages with `makedoc.g` benefit from it (but some care needs to be taken to ensure it doesn't break builds, e.g. by silly stuff, like requiring LaTeX in VMs that don't have it).

**Review by james-d-mitchell — APPROVED** [review]

**Comment by james-d-mitchell** [comment]
Thanks @fingolfin I'm happy with this too, I agree it'd be good to have a CI build targeting the documentation, will be happy to accept a PR on this basis.

**Pull request merged**
